### PR TITLE
feat: Update footer

### PR DESCRIPTION
**Pull request by Manav2304**

### CODE REVIEW

File: src/Components/Footer/style.ts [hunk 1]
```diff
@@ -17,24 +17,48 @@ export const Heading = styled.h6`
...
+
+  i {
+    margin-right: 4px;
+  }
+
+  span {
+    margin-left: 10px;
+    text-align: left;
+  }
+
```
Comment (vaidehi27) [inline src/Components/Footer/style.ts]: why nesting?

Comment (Manav2304) [reply src/Components/Footer/style.ts]: can i use >(child) selector instead of nesting

Comment (vaidehi27) [reply src/Components/Footer/style.ts]: why do you want to use any kind of nesting?

Comment (Manav2304) [reply src/Components/Footer/style.ts]: please check latest update I replace nesting with separate style component

**Review by vaidehi27 — APPROVED** [review]

**Review by vaidehi27 — APPROVED** [review]

**Pull request closed without merging**